core/src/com/ericc/the/game/Engines.java

```diff
@@ -75,4 +79,13 @@ public void removeEntity(Entity entity) {
     public ImmutableArray<Entity> getEntitiesFor(Family family) {
         return engine.getEntitiesFor(family);
     }
+
+    public void disableFieldOfViewSystem() {
```

Comment:
może to jest odrobinę ładniejsze, ale problem nadal pozostaje

Reply:
to się napisało do złej linijki, chodziło mi o special case na rejestrowanie FogSystem i FovSYstem